src/web/utils/urlHelpers.ts

```diff
@@ -0,0 +1,31 @@
+export const parseParticipantId = (participantId: string | undefined) => {
+  if (!participantId) {
+    throw new Error('Participant ID is not defined');
```

Comment:
is there a reason that we're expecting undefined participant ids here?  And if we're expecting them, is there a better way of dealing with that than just throwing?

Reply:
it should only happen if the user manually deletes the participantId from the URL, hence why I'm just throwing here. does that seem reasonable?

Reply:
sounds good.